qa/common/sequence_util.py

```diff
@@ -1,4 +1,6 @@
-# Copyright 2019-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#!/usr/bin/env python3
```

Comment:
Still don't totally understand. Did you manually add these shebangs? Or precommit complained about it specifically? Some of these utils probably don't need these nor to be executable - but not super important imo